tests/test_api.py

```
@@ -3124,31 +3124,27 @@ def test_list_job_templates_returns_created() -> None:
 # Issue #47 – Spark History Server link
 # ---------------------------------------------------------------------------
 
-def test_run_response_includes_spark_history_url() -> None:
+def test_run_response_drops_dead_spark_history_url_field() -> None:
+    """The dead ``spark_history_url`` response field (built from the always-NULL
+    ``spark_history_server_url`` env column) is gone. On-demand Spark UI access now
+    flows through ``GET /v1/runs/{id}/spark-ui`` (per-engine, minted, never stored).
+    """
     client = TestClient(app)
     _, op, job, run = _create_ready_environment_and_run(client, suffix="hist1")
```

Ruff (0.15.15)

[warning] Unpacked variable `job` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)